store/primary/cid/cid.go

```diff
@@ -92,7 +92,9 @@ func (cp *CIDPrimary) Get(blk types.Block) (key []byte, value []byte, err error)
 		return
 	}
 	read := make([]byte, CIDSizePrefix+int(blk.Size))
-	cp.file.ReadAt(read, int64(blk.Offset))
+	if _, err := cp.file.ReadAt(read, int64(blk.Offset)); err != nil {
+		return nil, nil, err
+	}
```

Comment:
It would be nice if the returns were consistent in the function.  Either assign the return values and then use a naked `return` or get rid of the named returns and return three values in each return.  I prefer the later.